include/index.js

```diff
@@ -214,6 +216,7 @@ module.exports = function PB(config) {
     pb.UrlService         = require(path.join(config.docRoot, '/include/service/entities/url_service.js'))(pb);
     pb.JobService         = require(path.join(config.docRoot, '/include/service/entities/job_service.js'))(pb);
     pb.TokenService       = require(path.join(config.docRoot, '/include/service/entities/token_service.js'))(pb);
+    pb.TokenServiceV2     = require(path.join(config.docRoot, '/include/koa/authentication/token_service.js'))(pb);
```

Comment:
Perhaps the name of the variable should be TokenServiceKOA, or KOATokenService.
Just having TokenServiceV2 may be confusing to someone down the line that has to make changes to this file.